src/org/etools/j1939_84/controllers/part01/Part01Step14Controller.java

```diff
@@ -103,6 +103,9 @@ protected void run() throws Throwable {
                                  && dm26System.getStatus().isComplete();
 
                      })
+                     .filter(dm26System -> {
+                         return getEngineModelYear() < 2019 && dm26System.getId() != CompositeSystem.MISFIRE;
```

Comment:
I don't think this logic is correct. But to prove me wrong, write a test that complains about misfire for MY 2019 and then doesn't complain when the MY is 2018